hw1/02-palindrome.html

```diff
@@ -6,20 +6,21 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
+  <body class="bg-primary">
+    <main class="container mt-5">
+      <section class="bg-light border rounded p-4">
+        <h1 class="card-title mb-4">Exercise 02 - Palindrome</h1>
+        <div class="mb-3">
           <label>Enter a positive number:</label>
-          <input type="number" />
+          <input id="number" type="number" />
```

Comment:
WAVE extension flagged the label for not being associated with a form input and the input was flagged for not having a label.